### PR TITLE
[WIP] Remove demo code

### PR DESCRIPTION
**Pull request by Kmaschta**
TODO:
- Remove API migrations
- Remove clear command itself
- Split `clear-demo-code` command into multiple commands

### CODE REVIEW

File: makefile [hunk 1]
```diff
@@ -152,3 +152,13 @@ create-admin:
 create-client:
 	# TODO: ensure we create a simple user and not an admin
 	./node_modules/babel-cli/bin/babel-node.js ./bin/createAdmin.js ${CLIENT_NAME} ${CLIENT_EMAIL} ${CLIENT_PASSWORD}
+
+clear-demo-code:
+	# Remove git origin
+	# git remote rm origin
```
Comment (djhi) [inline makefile]: Not sure about this

Comment (djhi) [inline makefile]: I think the `remote` is not needed

Comment (djhi) [inline makefile]: nvm, I'm wrong

**Comment by Kmaschta** [comment]
Needs #20 to be merged to remove frontend demo.

**Comment by fzaninotto** [comment]
Needs rebase

**Comment by Kmaschta** [comment]
Outdated as heck

**Pull request closed without merging**